docs-ui/stories/components/sidebarSection.stories.js

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+
+import {KeyValueTable, KeyValueTableRow} from 'app/components/keyValueTable';
+import {Body, Main, Side} from 'app/components/layouts/thirds';
+import QuestionTooltip from 'app/components/questionTooltip';
+import SidebarSection from 'app/components/sidebarSection';
+import Tag from 'app/components/tag';
+import TextOverflow from 'app/components/textOverflow';
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+
+export default {
+  title: 'Views/Sidebar Section',
+};
+
+export const Default = () => (
+  <Body>
+    <Main />
+    <Side>
+      <Container>
+        <SidebarSection title="Left Group">{'16 hours'}</SidebarSection>
+        <SidebarSection title="Right Group">{'16 hours'}</SidebarSection>
+      </Container>
+      <SidebarSection title="Subheading">{'16 hours'}</SidebarSection>
+      <SidebarSection
+        title="Icon, Tag, and Text"
+        icon={<QuestionTooltip position="top" title="Tooltip description" size="sm" />}
+      >
+        <div>
+          <Tooltip title="Tooltip description" isHoverable>
+            <Tag type="default">{'Adopted'}</Tag>
+          </Tooltip>
+          <Environment>{'in production'}</Environment>
+        </div>
+      </SidebarSection>
+      <SidebarSection title="With Rows">
+        <KeyValueTable>
+          <KeyValueTableRow
```

Comment:
I think this is a good example if you explained that another common use-case of using this component is with the `<KeyValueTable>`